helm/util/values.yaml

```diff
@@ -0,0 +1,32 @@
+elasticsearch:
+  internal: true
+  persistence:
+    enabled: false
+  replicas: 1
+  minimumMasterNodes: 1
+  resources:
+    requests:
+      memory: 1Gi
+kibana:
+  internal: true
+  env:
+    ELASTICSEARCH_URL: http://elasticsearch-client:9200
+fluentbit:
+  enabled: true
+  backend:
+    type: es
+    es:
+      host: localhost
+      port: 65535
```

Comment:
I thought this was supposed to be the default hostname and port for the internal ES pod/service